```
F:\workspace\_work\1\s\src\Identity\EntityFrameworkCore
 \test\EF.Test\Utilities\ScratchDatabaseFixture.cs(23,28): 
error CS1061: 'Lazy<SqlServerTestStore>' does not contain a definition for 'HasValue'
 and no accessible extension method 'HasValue' accepting a first argument of type 
  'Lazy<SqlServerTestStore>' could be found (are you missing a using directive or an assembly reference?) 
[F:\workspace\_work\1\s\src\Identity\EntityFrameworkCore\test\EF.Test\Microsoft.AspNetCore.Identity.EntityFrameworkCore.Test.csproj]
```

--- a/src/Identity/EntityFrameworkCore/test/EF.Test/Utilities/ScratchDatabaseFixture.cs
+++ b/src/Identity/EntityFrameworkCore/test/EF.Test/Utilities/ScratchDatabaseFixture.cs
@@ -20,7 +20,7 @@ namespace Microsoft.AspNetCore.Identity.EntityFrameworkCore.Test
 
         public void Dispose()
         {
-            if (_testStore.HasValue)
+             if (_testStore.IsValueCreated)
             {
                 _testStore.Value.Dispose();
             }

